lib/sync/index.js

```diff
@@ -46,12 +46,13 @@ var syncLock = null;
 var interceptors = null;
 var apiSync = null;
 var apiSyncRecords = null;
+var syncScheduler = null;
```

Comment:
This gets initialized in `.start`, and accessed in `.stopAll`, so it should be declared here for both.